dataCollection/sensors/PicoColorMatcher.java

```diff
@@ -60,6 +62,11 @@ private Color getTargetColor() {
 
   public PicoColorMatcher() {
     init();
+    COLORMATCHER_RED = "ColorMatcher/Red"; 
```

Comment:
should have some kind of differentiator if we have multiple of these. or is that not possible with the current pico implementation? if not, then ok

Reply:
bump

Reply:
We can differentiate these. (Line 170 PicoColorSensor) int port = SerialPortJNI.serialInitializePort((byte)1);
We should be able to differentiate by passing a port in to the methdo

Reply:
Also, I'll make this change on 2024-updated

Reply:
Ok. Add a trello card so we don't forget